doc/compiled.json

```diff
@@ -1308,7 +1308,38 @@
       "project_locales": {
         "allOf": [
           {
-            "$ref": "#/components/schemas/project_short"
+            "type": "object",
+            "title": "project_member_specific",
+            "properties": {
+              "id": {
+                "type": "string"
+              },
+              "name": {
+                "type": "string"
+              },
+              "project_role": {
+                "type": "string"
+              },
+              "main_format": {
+                "type": "string"
+              },
+              "created_at": {
+                "type": "string",
+                "format": "date-time"
+              },
+              "updated_at": {
+                "type": "string",
+                "format": "date-time"
+              }
+            },
+            "example": {
+              "id": "abcd1234cdef1234abcd1234cdef1234",
+              "name": "My Android Project",
+              "project_role": "Developer",
+              "main_format": "xml",
+              "created_at": "2015-01-28T09:52:53Z",
+              "updated_at": "2015-01-28T09:52:53Z"
+            }
```

Comment:
added by accident?

Reply:
the autogenerated output was just not added by a previous PR and I thought it makes sense to keep it consistent